vapi/appliance/simulator/simulator.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package simulator
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/vapi/appliance/access/consolecli"
+	"github.com/vmware/govmomi/vapi/appliance/access/dcui"
+	"github.com/vmware/govmomi/vapi/appliance/access/shell"
+	"github.com/vmware/govmomi/vapi/appliance/access/ssh"
+	vapi "github.com/vmware/govmomi/vapi/simulator"
+)
+
+func init() {
+	simulator.RegisterEndpoint(func(s *simulator.Service, r *simulator.Registry) {
+		New(s.Listen).Register(s, r)
+	})
+}
+
+// Handler implements the Appliance API simulator
+type Handler struct {
+	URL        *url.URL
+	consolecli struct {
+		access consolecli.Access
```

Comment:
since you're using the defaults here, could you

a) add a note that the values are disabled/0 by default or 
b) explicitly set the values so it's clear what the default behavior is?

Reply:
now setting the values explicitly